webhooks/cluster_test.go

```diff
@@ -443,3 +540,66 @@ func TestClusterTopologyValidation(t *testing.T) {
 		})
 	}
 }
+
+// TestValidateCreate tests the additional cases introduced in new validation in the webhook package.
+func TestValidationsUsingClient(t *testing.T) {
```

Comment:
```suggestion
func TestValidationsUsingClient(t *testing.T) {
```
nit, maybe: `TestClusterTopologyValidationWithClient` or `TestClusterTopologyValidationWithClusterClass` (for more consistency with existing func names)